projects/kit/components/input-slider/input-slider.template.html

```diff
@@ -2,7 +2,7 @@
     [appearance]="appearance"
     [readOnly]="readOnly"
     [disabled]="disabled"
-    [focused]="computedFocused"
+    [focused]="computedFocused || sliderFocused"
```

Comment:
why the `sliderFocused` getter can't it be moved to the `computedFocused` getter?

Reply:
oh no, sorry, my last message is not true.

It doesnt update value while drag by another reason.
I'll add a commit with changes

Reply:
@splincode I have changed it now as you proposed.

Now `InputSlider` should work in the same way as it was.